src/isar_exr/config/settings.py

```diff
@@ -21,6 +21,10 @@ class Settings(BaseSettings):
         default="https://login.energy-robotics.com/api/loginApi"
     )
 
+    # API variables
+    API_SITE_ID: str = Field(default="6287873b6cc0c0205c56ea58")
+    API_ROBOT_ID: str = Field(default="628787246cc0c0205c56e88e")
```

Comment:
I think this is already defined on line 9

Reply:
Updated value on line 9